docs/refman/generatorman.py

```diff
@@ -24,8 +24,10 @@ def reset_font(self) -> None:
 
     def title(self, name: str, section: int) -> None:
         import datetime
+        import time
+        from os import environ
```

Comment:
I don't think there's any reason to be importing these inside of the `title` function. Importing inside a method is generally a code smell related to spaghetti code, not an optimization method.

I'd also prefer to just call `os.environ`, `os` isn't that long of a name.

Other than that, this looks good to me. And with those things changed I'm happy to see this land

Reply:
> I don't think there's any reason to be importing these inside of the `title` function. Importing inside a method is generally a code smell related to spaghetti code, not an optimization method.

I too think importing stuff from functions looks odd, but I simply tried to follow what was already being done by the script. Should I also move the `datetime` import at the top level?

> I'd also prefer to just call `os.environ`, `os` isn't that long of a name.

Got it, will fix it soon.

Reply:
> I too think importing stuff from functions looks odd, but I simply tried to follow what was already being done by the script. Should I also move the `datetime` import at the top level?

I think so, yes. The datetime import was never intentionally there to begin with -- it's been that way since @annacrombie initially PR'ed the man page backend. In other refman backends, we do imports inside a method when the import is a third-party package, and we include comments such as:
```
# Import here, so that other generators don't also depend on it
```